README.md

```diff
@@ -132,18 +213,19 @@ Install the repository into `idrstream/` with:
 cd idrstream/
 git clone https://github.com/cytomining/DeepProfiler.git
 cd DeepProfiler/
+# make sure that the conda environment `idrstream_dp` is activated
 pip install -e .
 ```
 
 ## Example Usage
 
-Example usage of `idrstream` can be found at [example.ipynb](example.ipynb).
-We ran this notebook as a python script ([example.py](example.py)) and hence the notebook has no output for the final cell.
+Example usage of `idrstream` can be found at [example_dp.ipynb](example_dp.ipynb) and [example_cp.ipynb](example_cp.ipynb).
+We ran this notebook as a python script ([example_dp.py](example_dp.py) and [example_cp.py](example_cp.py)) and hence the notebook has no output for the final cell.
 
-**Note**: You can use `idrstream` o extract object oulines as extra metadata by passing `extra_metadata=["object_outlines"]` during `idrstream.run_stream()`.
+**Note**: You can use `idrstream` to extract object oulines as extra metadata by passing `extra_metadata=["object_outlines"]` during `idrstream.run_stream()`.
```

Comment:
Just want to note there that this is not true for this PR. However, I think this note should be left because the plan is to add this functionality in the next PR.

Reply:
True, I agree to just leave it.